### ISSUE TITLE
What is the scope of this project?

### ISSUE THREAD

**Issue by alecgorge** [monicahq/rachel#1]
I stumbled across the repo and I'm very excited about possibilities here!

What is the scope/goal of this project? Will it be able to pull in friend's life events?

What is the motivation to do it as a chrome extension instead of a Facebook API integration in the webapp itself?

**Comment by Mokto** [comment]
Hi,

Sorry for the late reply ! We are focusing on the mobile app right now, that's why there hasn't been any commit for a long time.

The main motivation behind this project is to easily import a contact (avatar, first name, last name, birthdate, ...). It is not possible to use the Facebook API to get this information.

But eventually we could add a lot a handy functionalities (open for discussion).

**Comment by djaiss** [comment]
@alecgorge FB's API does not give any information about the contact (and rightfully so, as it's the core of their business).

As Mokto said, the idea is to use this extension to "scrape" a profile page and retrieve the information needed to add a contact on Monica.

**Comment by cweagans** [comment]
> FB's API does not give any information about the contact (and rightfully so, as it's the core of their business).

I don't think this is accurate. I was able to pull a substantial amount of information from Facebook's API about my friends (at least First & Last name, birthday, email address, etc for those that have that information populated and visible to me). Here's the full list of fields that you can retreive: https://developers.facebook.com/docs/graph-api/reference/user

What can't you get from that API that you need? Screen scraping is going to be slow in comparison, and you're not going to be able to get any information that way that you cannot get from the API.

**Comment by ishmaelthedestroyer** [comment]
Hi! Would anyone be interested in working on this? I'd love to bring some life into this project.
